modules/ROOT/partials/import-database.adoc

```diff
@@ -22,7 +22,10 @@ Once this has finished, the instance returns to its `Running` state; and the dat
 
 [CAUTION]
 ====
-This command does not currently support xref:platform/security.adoc#_vpc_isolation[private linking]. Please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] if you have public traffic disabled and need to use this command.
+If you have enabled xref:platform/security.adoc#_vpc_isolation[private linking], you must ensure public traffic is enabled for the region your instance is hosted in before you can use this command.
+
+Public traffic can be enabled from the *Network access* section of the Console by selecting *Configure* on the region you want to change and un-ticking the *Disable public access* option in step 4.
```

Comment:
Maybe "deselecting" is better than "un-ticking"?

Reply:
Microsoft suggests 'clear' but i wonder if that is even more confusing - https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/deselect

Reply:
Same for "unchecking": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/uncheck-unmark-unselect. I agree it may be more confusing too.

How about rephrasing as the following:

```suggestion
You can enable public traffic from the *Network access* section of the Console by selecting *Configure* on the region you want to change and making sure the *Disable public access* option in step 4 is not selected.
```